packages/cli/__tests__/__mocks__/snapshot/pages/home.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
```

Comment:
Are these files mock snapshots? (File path is `__mocks__/snapshot`)

Reply:
These are files I created by manually running __mocks__/source through the CLI in order to use in my tests to validate the output.